src/api/BlogPosts.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable camelcase */
 import { WP_REST_API_Post } from 'wp-types';
-type ApiPost = WP_REST_API_Post;
+
+type ApiPost = WP_REST_API_Post & {
+	pg_preview_link: string;
```

Comment:
We could rename this to something not specific to Playground, as this would also apply in a scenario where we're not using Playground for the preview.